src/components/Layout/UserDropdown/index.tsx

```diff
@@ -55,6 +58,36 @@ const UserDropdown: React.FC = () => {
             aria-orientation="vertical"
             aria-labelledby="user-menu"
           >
+            <Link href={`/profile`}>
+              <a
+                className="block px-4 py-2 text-sm text-gray-200 transition duration-150 ease-in-out hover:bg-gray-600"
+                role="menuitem"
+                tabIndex={0}
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter') {
+                    setDropdownOpen(false);
+                  }
+                }}
+                onClick={() => setDropdownOpen(false)}
+              >
+                <FormattedMessage {...messages.myprofile} />
```

Comment:
```suggestion
                {intl.formatMessage(messages.myprofile)}
```

I thought that we should be using `intl.formatMessage()` going forward?

Reply:
Not sure why I used the component here. 🤔